verilog/formatting/tree_unwrapper.cc

```diff
@@ -788,6 +787,26 @@ void TreeUnwrapper::Visit(const verible::SyntaxTreeNode& node) {
       break;
     }
 
+    case NodeEnum::kTaskHeader: {
```

Comment:
Want to include `kFunctionHeader` too?

Reply:
Yes, I do want to include `kFunctionHeader`. Done in https://github.com/google/verible/pull/148/commits/7fa1fe5740f8c73b756ac670a34c54afa33960c9